src/vcpkg/commands.acquire-project.cpp

```diff
@@ -1,15 +1,26 @@
 #include <vcpkg/base/checks.h>
+#include <vcpkg/base/util.h>
 
 #include <vcpkg/commands.acquire-project.h>
 #include <vcpkg/configure-environment.h>
 #include <vcpkg/vcpkgcmdarguments.h>
 
 namespace vcpkg
 {
+    const CommandMetadata CommandAcquireProjectMetadata{
+        [] { return create_example_string("acquire-project"); },
+        0,
+        0,
+        {CommonAcquireArtifactSwitches, {}, {}},
```

Comment:
I think this results in a global constructor because `CommonAcquireArtifactSwitches` isn't constexpr.

Could we move `CommonAcquireArtifactSwitches` into a separate private header and make it constexpr, so that `CommandAcquireProjectMetadata` etc can also be made constexpr?

Reply:
Yeah I guess potentially duplicating the literals is less bad than dynamic initializers.